documentation/APIs/openapi3-core.yaml

```diff
@@ -2570,7 +2694,7 @@ components:
             - decimals
             - useMetricPrefix
         features:
-          description: The features that are supported by the node. For example, a node could support the Proof-of-Work (pow) feature, which would allow the Proof-of-Work to be performed by the node itself. All features must be lowercase.
+          description: The features that are supported by the node. For example, a node could support the feature, which would allow the BIC to be included by the node account. All features must be lowercase.
```

Comment:
I'm not sure if this is accurate. I'd leave out the example for now.

Reply:
Currently, we have a `allowIncompleteBlock` in features (a string slice defined in core-api component), if it's enabled, the node will fill the incomplete block, resign then issue it and of course will consume its BIC.

Reply:
Didn't know about that, then I'll retract my statement :smile: